src/partials/header.html

```diff
@@ -0,0 +1,18 @@
+<header class="header container">
+  <div class="header__container">
+    <a href="#" class="header__logo">
+      <img
+        src="./images/logo/Logo-l-min.png"
```

Comment:
Please, use example.html to find out the right way in the src attribute.

Reply:
got it! done!